packages/ui/components/apps/AppCard.tsx

```diff
@@ -170,3 +167,103 @@ export function AppCard({ app, credentials, searchText }: AppCardProps) {
     </div>
   );
 }
+
+const InstallAppButtonChild = ({
+  userAdminTeams,
+  addAppMutationInput,
+  appCategories,
+  credentials,
+  ...props
+}: {
+  userAdminTeams?: UserAdminTeams;
+  addAppMutationInput: { type: App["type"]; variant: string; slug: string };
+  appCategories: string[];
+  credentials?: Credential[];
+} & ButtonProps) => {
+  const { t } = useLocale();
+  const router = useRouter();
+
+  const mutation = useAddAppMutation(null, {
+    onSuccess: (data) => {
+      // Refresh SSR page content without actual reload
+      router.replace(router.asPath);
+      if (data?.setupPending) return;
+      showToast(t("app_successfully_installed"), "success");
+    },
+    onError: (error) => {
+      if (error instanceof Error) showToast(error.message || t("app_could_not_be_installed"), "error");
+    },
+  });
+
+  if (
+    !userAdminTeams?.length ||
+    appCategories.some(
+      // We can remove these as we allow more of these installs
+      (category) => category === "calendar" || category === "video"
```

Comment:
As per [this](https://github.com/calcom/cal.com/blob/d051a54aa91d4a34f96680405a18dfba39f04a82/apps/web/components/apps/App.tsx#L101) only `calendar` category allows multiple install.

Reply:
Sorry this was an artifact comment. For orgs right now we don't want calendars and video apps to be installed team wide.